lib/response/set/SetResponse.js

```diff
@@ -84,7 +84,7 @@ SetResponse.prototype._subscribe = function _subscribe(observer) {
 
     // Starts the async request cycle.
     return setRequestCycle(
-        model, observer, groups, isJSONGraph, isProgressive, 0);
+        model, observer, groups, isJSONGraph, isProgressive, 1);
```

Comment:
@tquetano-netflix The initial value was changed here, in coordination with the retry logic condition.